modules/identity-provider-config-map.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * authentication/identity_providers/configuring-basic-authentication-identity-provider.adoc
+// * authentication/identity_providers/configuring-github-identity-provider.adoc
+// * authentication/identity_providers/configuring-gitlab-identity-provider.adoc
+// * authentication/identity_providers/configuring-ldap-identity-provider.adoc
+// * authentication/identity_providers/configuring-oidc-identity-provider.adoc
+// * authentication/identity_providers/configuring-request-header-identity-provider.adoc
+
+[id='identity-provider-creating-configmap-{context}']
+= Creating a ConfigMap
+
+Identity providers use {product-title} ConfigMaps to contain the 
+certificate authority bundle.  
+
+* Define an {product-title} ConfigMap containing a the
+certificate authority by using the following command.
++
+----
+$ oc client configmap ca-config-map --from-file=/path/to/ca
```

Comment:
oc client? maybe a typo! 
And using key **ca.crt**:
`oc create configmap ca-config-map --from-file=ca.crt=/path/to/ca`

Reply:
The `oc client` commands have been updated to say `oc create`. In addition, the ConfigMap command now uses your text.

The PR has also been rebased.